README.md

```diff
@@ -64,64 +21,129 @@ public class App {
+
+# HTTP Server Local Development Setup
+### 1. Install Java
+Check if Java is present: `java -version`
+
+This project is currently running `java version "12.0.1" 2019-04-16`.
+If not present, follow these [instructions](https://www.notion.so/Setting-Up-Java-Environment-1a48792fb5c6403bbb430c882e411226#3b7fec7b6e6d4f06a82dca4afcf31081).
 
-app.get("/get_with_body", (Request request, Response response) -> {
-        res.body("Here are all my favorite movies:\n" + "- Harry Potter\n");
-});
+### 2. Install Gradle
+`brew install gradle`
+Verify you have it installed with `gradle -v`
+
+### 3. Clone repo
+`git clone https://github.com/serpient/java_http_server.git`
+
+### 4. Run Build or Test
+Within project folder, run `./gradlew build` to see the status
+
+Within project folder, run `./gradlew test` to run the tests
+
+### 5. Run the server from project directory
+**Option 1: Default port 5000**
+```
+gradle run
 ```
-### Request and Response objects are available in the route callback.
-* Request allows users access to the request method, body and route from the callback.
-* Response allows users to pre-define the status, add custom headers, and edit the response body.
 
-# Testing Multiple Requests
-- To test the simultaneous client sessions with a benchmark, start App.main() and then run `ab -r -n 1000 -c 50
-http://localhost:5000/` in another terminal.
+**Option 2: Custom port and valid directory**
+```
+ gradle run --args="-p 3000 -d /test"
+```
 
-# Testing with Postman
-### Install Postman
-### Open terminal, navigate to project directory, then run `gradle run`
-### Open Postman and run the different routes currently available:
+**Option 3: Custom port and valid directory from jar**
 ```
-GET /simple_get
-HEAD /simple_get
-GET /get_with_body
-HEAD /get_with_body
-POST /echo_body
+gradle build
+java -jar build/libs/application.jar -p 2000 -d /files
 ```
-# Testing with the Acceptance Tests
-### Clone the [acceptance tests repository](https://github.com/8thlight/http_server_spec) and follow install instructions
 
-### Open terminal, navigate to project directory, then run `gradle run`
-### Open terminal, navigate to acceptance tests directory, then run `bundle exec spinach`
\ No newline at end of file
+**Arguments**
+- `-p` for Port
+- `-d` for Directory path
+
+
+# Testing
+#### Testing Multiple Requests
+- To test simultaneous client sessions with a benchmark, start App.main() and then run `ab -r -n 1000 -c 50
+http://localhost:5000/` in another terminal.
+
+#### Testing with Postman
+1. Install Postman
+2. Open terminal, navigate to project directory, then run `gradle run`
+3. Open Postman and run the different routes defined in the Application:
+
+#### Testing with the Acceptance Tests
+1. Clone the [acceptance tests repository](https://github.com/8thlight/http_server_spec) and follow install
```

Comment:
I think you should link to the acceptance test repo with the extra tests that you added instead of the 8th light one